packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -240,6 +245,7 @@ export const Combobox = ({
           maxWidth: popoverMaxWidth,
           minWidth: inputRef.current?.offsetWidth,
           width: popoverWidth,
+          display: isOpen ? 'block' : 'none', // Remove empty list box from screen reader when closed
```

Comment:
We should move this to emotion styles. I would probably prefer a new class name for `styles.menuOpen` or something.